docs/generated/oc_by_example_content.adoc

```diff
@@ -920,6 +933,32 @@ Create a service account with the specified name.
 ====
 
 
+== oc create user
+Create a user.
+
+====
+
+[options="nowrap"]
+----
+  # Create a user with the username "ajones" and the display name "Adam Jones"
+  $ oc create user ajones --full-name="Adam Jones"
+----
+====
+
+
+== oc create useridentitymapping
```

Comment:
These are not for Joe User... only a cluster admin would have the ability to even run these. I would expect them to use these in a script to provision a user one piece at a time.